src/datacenters/v1/README.md

```diff
@@ -0,0 +1,216 @@
+# v1 datacenter schema
+
+The v1 datacenter schema is designed to allow operators to control resources in three
+different scopes:
+
+- Datacenter-scoped resources
+- Environment-scoped resources
+- Application-scoped resources
+
+By combining these three scopes, operators can create robust datacenter templates that
+can be used to host and integrate virtually any application in any style of cloud
+environment.
+
+## Variables
```

Comment:
I think this section kind of jumps the gun. It is a nice feature but only once you really understand what is happening.